src/parser/jobs/ast/modules/ArcanaSuggestions.js

```diff
@@ -2,28 +2,35 @@ import {t} from '@lingui/macro'
 import {Trans} from '@lingui/react'
 import _ from 'lodash'
 import React, {Fragment} from 'react'
-import {Table, Icon, Message, Label, Accordion} from 'semantic-ui-react'
+import {Table, Icon, Message, Label, Accordion, Button} from 'semantic-ui-react'
 
-import {ActionLink} from 'components/ui/DbLink'
-import JobIcon from 'components/ui/JobIcon'
+import {DRAWN_ARCANA_USE, HELD_ARCANA_USE, SPEAR_USED} from './ArcanaGroups'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+import Module from 'parser/core/Module'
 import ACTIONS, {getAction} from 'data/ACTIONS'
+import JobIcon from 'components/ui/JobIcon'
 import JOBS from 'data/JOBS'
-import Module from 'parser/core/Module'
-import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {ActionLink} from 'components/ui/DbLink'
+import styles from './ArcanaSuggestions.module.css'
 
-import {DRAWN_ARCANA_USE, HELD_ARCANA_USE, SPEAR_USED} from './ArcanaGroups'
-import DISPLAY_ORDER from './DISPLAY_ORDER'
+const LADY_OF_CROWNS_STATUS_ICON = 'https://xivapi.com/i/014000/014840.png'
+const LORD_OF_CROWNS_STATUS_ICON = 'https://xivapi.com/i/014000/014841.png'
+const UNKNOWN_CARD_STATUS_ICON = 'https://xivapi.com/i/010000/010205.png'
```

Comment:
Guessing these are the PVP icons or similar?

Reply:
Yeah they look more consistent